### PR TITLE
Add playbooks and runs in sidebar

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This PR adds playbooks and runs in sidebar. List of playbooks and list of runs are not "correct", will be changed in the next PRs. This is just for UI testing.

https://user-images.githubusercontent.com/2340127/173399634-bcef977d-83fc-425b-a44b-65aaf6e4966d.mp4

#### Checklist

- [ ] Telemetry updated
- [ ] Gated by experimental feature flag
- [ ] Unit tests updated

### CODE REVIEW

**Comment by abhijit-singh** [comment]
Thanks @iomodo, awesome progress! These icons aren't in the compass icon font yet but I'll work on getting them in this week and then ideally we should be using those. This can be done in a future PR and not necessarily right now. Would that be okay?

**Comment by iomodo** [comment]
Wonderful @abhijit-singh, thanks!

File: webapp/src/components/sidebar/playbooks_sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    ${sharedIconStyles}
+`;
+
+const StyledPlaybookRunIcon = styled(PlaybookRunIcon)`
+    ${sharedIconStyles}
+`;
+
+const StyledPrivatePlaybookIcon = styled(PrivatePlaybookIcon)`
+    ${sharedIconStyles}
+`;
```
Comment (trilopin) [inline webapp/src/components/sidebar/playbooks_sidebar.tsx:133]: nit: newline missing

**Review by trilopin — APPROVED** [review]
Nice job @iomodo!

just some tiny nits about the copyright comment (some files had with bad year, some had not).

Question: sidebar and LHS names are interchangeable or do they have different purposes?

**Review by abhijit-singh — APPROVED** [review]
Looks good!

File: webapp/src/components/sidebar/playbooks_sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+export default PlaybooksSidebar;
+
+const sharedIconStyles = css`
+    width: 18px;
+    height: 18px;
+`;
+
+const StyledPlaybookIcon = styled(PlaybookIcon)`
+    ${sharedIconStyles}
+`;
```
Comment (calebroseland) [inline webapp/src/components/sidebar/playbooks_sidebar.tsx:125]: For future pending icons, using the MDI version directly might be simpler

```suggestion
const StyledPlaybookIcon = () => (
    <Icon
        path={mdiBookOutline}
        size={1.25}
    />
);
```

File: webapp/src/components/sidebar/sidebar.tsx [hunk 3]
```diff
@@ -60,7 +63,15 @@ const Sidebar = (props: SidebarProps) => {
                 </OverlayTrigger>
                 {props.headerDropdown}
             </Header>
-            <Groups>
+            <Scrollbars
```
Comment (calebroseland) [inline webapp/src/components/sidebar/sidebar.tsx:66]: There's a slight gap next to the scrollbar

CleanShot 2022-06-14 at 22 45 16

**Review by calebroseland — APPROVED** [review]
Looking great so far!

File: webapp/src/components/assets/icons/private_playbook_icon.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
```
Comment (calebroseland) [inline webapp/src/components/assets/icons/private_playbook_icon.tsx:1]:
```suggestion
// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

File: webapp/src/components/sidebar/playbooks_sidebar.tsx (hunk 1 above)
Comment (calebroseland) [inline webapp/src/components/sidebar/playbooks_sidebar.tsx:133]:
```suggestion
`;

```

**Pull request merged**